app/src/main/java/com/deveddy/clujbike/data/repository/mapper/DataMapper.java

```diff
@@ -0,0 +1,5 @@
+package com.deveddy.clujbike.data.repository.mapper;
+
+public interface DataMapper<T, V> {
+    V mapFrom(T t);
```

Comment:
What if we would change the names here? I think that the following would be simpler and cleaner:
```java
public interface Mapper<F, T> {
    T from(F from);
}
```
with `T` from `to` and `F` from `from`.
What do you think?